src/reporter.ts

```diff
@@ -176,14 +179,17 @@ export class Reporter {
       withoutIssue = this.config.skippedIssue === false;
     } else if (hasError) {
       status = STATUSES.FAILED;
+      const errorMsg = testRunInfo.errs[testRunInfo.errs.length - 1].errMsg;
```

Comment:
It would be better to have an equal error log in description and the last error log on the log page.
So I suggest to use error formating as in `sendLogsOnFail` method:
```javascript
const errorMsg = stripAnsi(this.formatError(testRunInfo.errs[testRunInfo.errs.length - 1])); 
```